src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+  position: relative;
+
+  font-size: 12px;
+  font-weight: $medium-font;
```

Comment:
It seems like you are using a variable `$medium-font` for the `font-weight` property. Ensure that this variable is defined in your `variables.scss` file and corresponds to one of the standard font-weight numeric values, such as 400, 500, etc.